### ISSUE TITLE
No Bellatrix preset in `/eth/v1/config/spec`

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#3388]
## Description

There's a `TODO(merge)` here which prevents Lighthouse from serving the Bellatrix presets on `/eth/v1/config/spec`:

https://github.com/sigp/lighthouse/blob/21dec6f603edd53ad8d2372941c77ab8098d3b5a/consensus/types/src/config_and_preset.rs#L18-L20

We should try re-enabling that preset and possibly removing all of the other defaults added in https://github.com/sigp/lighthouse/pull/2934.

We should be careful not to break compatibility with any recent versions of Lighthouse or the other clients. I _think_ the VC has become more liberal in what it will accept since that old compat PR was made, but we need to check.

Having dynamic logic to only serve the Bellatrix preset if the Bellatrix epoch is set is another (potentially higher effort) option

**Comment by TobiWo** [comment]
Hi @michaelsproul, while connecting the teku vc to the lighthouse bn I'm receiving the following error:
```
failed to retrieve network spec from beacon node endpoint 'http://lighthouse-beacon-prater:5052/'.
Details: Missing value for spec constant 'INACTIVITY_PENALTY_QUOTIENT_BELLATRIX'
```
This only happens when teku is running with `--network=auto`. So is this issue here the reason for the error I receive? If it is not I would open a new issue.

**Comment by michaelsproul** [comment]
@tobisako Yeah this issue is the cause of that. So Teku is usable if you specify the network manually?

**Comment by dB2510** [comment]
@TobiWo teku should work if you specify for example: `--network=goerli`

**Comment by TobiWo** [comment]
@michaelsproul Yes, if I specify the network manually, as stated by @dB2510, everything works fine.

**Comment by paulhauner** [comment]
Resolved via #3425 :tada:
